pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -182,9 +182,6 @@ def simulate_and_vjp(
 _operations = frozenset(
     {
         "Identity",
-        "BasisState",
-        "QubitStateVector",
-        "StatePrep",
```

Comment:
For the reference, these ops are also decomposed in Catalyst by default

https://github.com/PennyLaneAI/pennylane-lightning/blob/d0e1c51d9da35883ab9b9e6df0a1bdf94878102e/pennylane_lightning/lightning_qubit/lightning_qubit.toml#L72-L78